src/components/Filter.js

```diff
@@ -60,6 +60,7 @@ function Filter({
   useEffect(() => {
     // fetch all abilities
     getAllAbilities();
+    // eslint-disable-next-line
```

Comment:
Nice disabling of annoying ESlint!

Reply:
Yeah, I really wish eslint had the ability to recognize that the empty array at the end of useEffect is intentional and valid.

Reply:
Yeah...